sleap/gui/commands.py

```diff
@@ -656,6 +664,37 @@ def ask(context: "CommandContext", params: dict) -> bool:
         return True
 
 
+class ImportNWB(AppCommand):
+    @staticmethod
+    def do_action(context: "CommandContext", params: dict):
+
+        labels = Labels.load_nwb(filename=params["filename"])
+
+        new_window = context.app.__class__()
+        new_window.showMaximized()
+        new_window.loadLabelsObject(labels=labels)
+
+    @staticmethod
+    def ask(context: "CommandContext", params: dict) -> bool:
+        filters = ["NWB (*.nwb)"]
```

Comment:
Should really reference the `NXDPoseAdaptor` class here and use `NDXAdaptor.name` with `NDXAdaptor.all_exts`